Where_Is_My_Wife/Assets/_Utilities/_Ferchus/StringArray.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using UnityEngine;
+
+[Serializable]
+public class StringArray 
```

Comment:
Not needed

Reply:
96665c08c08aa3776ec96d7030a7f19c1066d17c